### ISSUE TITLE
Missing support for sudo insults

### ISSUE THREAD

**Issue by ricardobranco777** [microsoft/sudo#31]
These can be included and the license is compatible:

https://github.com/sudo-project/sudo/blob/main/plugins/sudoers/insults.h

**Comment by DHowett** [comment]
You've brought me back a memory of building my Gentoo system with `USE=+offensive`. Thanks. 🙂

**Comment by luxzg** [comment]
While certainly fun, I don't think MS will adopt any of these:
```
[sudo] password for xxx: 
The more you drive -- the dumber you get.
[sudo] password for xxx: 
I've seen penguins that can type better than that.
[sudo] password for xxx: 
This mission is too important for me to allow you to jeopardize it.
[sudo] password for xxx: 
He has fallen in the water!
[sudo] password for xxx: 
Hold it up to the light --- not a brain in sight!
```

;-D

**Comment by pixiekat** [comment]
While you're at it, give us support for `--no-preserve-root`.

**Comment by joadoumie** [comment]
I agree this would be fun, but for now I am going to move this into the 'Ice Box' milestone as we have other things we're going to prioritize over this.

Thanks for creating the issue and letting some of us relive some great moments -- I personally did not know about this feature until this issue, and I'm happy that I now do!

**Comment by Bratzmeister** [comment]
![image](https://github.com/microsoft/sudo/assets/13728692/aeeb829b-8aad-4504-817c-9718a3073fd7)
I know it's already insulting enough to use sudo on non-unix system but please let me feel it
